model/accounts_mgmt/v1/subscription_type.model

```diff
@@ -34,17 +34,25 @@ class Subscription {
 	DisplayName String
 	Managed Boolean
 	Status String
+	Provenance String
 	// Last time this subscription were reconciled about cluster usage
 	LastReconcileDate Date
 	ConsumerUUID String
+	ConsoleURL String
 	Labels []Label
-	Provenance String
+	Capabilities []Capability
+	// Last time status was set to Released for this cluster/subscription in Unix time
+	LastReleasedAt Date
+	Released Boolean
+	// Metrics array. One or zero elements will present
+	Metrics []SubscriptionMetrics
+	NotificationContacts []Account
 	CloudAccountID String
 	CloudProviderID String
 	RegionID String
 	ClusterBillingModel BillingModel
-	// Metrics array. One or zero elements will present
-	Metrics []SubscriptionMetrics
+	// If the subscription is a trial, date the trial ends
```

Comment:
Try to end documentation sentences with punctuation:

```
If the subscription is a trial, date the trial ends.
```

That punctuation is a good typist practice, and in addition we will probably use it in the future to extract the first sentence of the documentation to use it as a summary.